eng/ci/official-build.yml

```diff
@@ -43,6 +43,7 @@ extends:
       - template: /eng/ci/templates/official/jobs/build-artifacts.yml@self
 
     - stage: Test
+      dependsOn: [] # allows you to run stages in parallel
```

Comment:
I thought they already ran in parallel?

Reply:
Stages by default run in sequence, but jobs do run in parallel by default though